src/parser/jobs/smn/modules/Gauge.js

```diff
@@ -97,19 +92,14 @@ export default class Gauge extends Module {
 		}
 
 		if (AETHER_ACTIONS.includes(abilityId)) {
-			// Aether actions convert flow into trail
-			// TODO: Check for using flow when none (logic issue)
-			this._aetherflow --
-			this._aethertrailAttunement ++
+			if (this._aetherflow > 0) {
```

Comment:
conditionals don't need to be nested

might be worth raising a broken log if they're using aetherflow when we think we're at 0?